_ml-commons-plugin/pretrained-models.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Pretrained models
+parent: Model-serving framework
+nav_order: 120
+---
+
+# Pretrained models
+
+The model-serving framework supports a variety of open-source pretrained models that can assist with a range of machine learning (ML) search and analytics use cases. 
+
+## Uploading pretrained models
+
+To use a pretrained model in your OpenSearch cluster:
+
+1. Select the model and download the model artifact and config.json file. For a list of pretrained models, see [supported pretrained models](#supported-pretrained-models).
```

Comment:
After `Select the model and download the model artifact and config.json file.` can we refer to the section `Sentence transformers`, so that customers don't get confused which models they need to download.

Reply:
@dhrubo-os: Since the plan is to add more models in the future, I would rather link to the general section.